scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +657,105 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def __getitem__(self, val):
+        # Returns a single rotation for an index, and a stack of rotation(s)
+        # for slice, index array, etc even if only one index is ultimately
+        # given
+        return self.__class__.from_quaternion(self._quat[val], normalized=True)
+
+    def copy(self):
+        return self.__class__.from_quaternion(
```

Comment:
I think you might be mistaken that `__init__` actually does copying, when `normalized=True`.

Reply:
I'll look into it.

Reply:
You're right, `__init__` doesn't actually copy when `normalized==True`. If any modifications are made to the input quaternion array later on in the code, they are reflected in the `self._quat` property as well. Should we be copying the quaternions in `__init__` itself to ensure such things cannot happen? This would lead to `__getitem__` also always returning new objects.